packages/renderer/src/lib/statusbar/StatusBarItem.svelte

```diff
@@ -56,7 +56,7 @@ async function executeCommand(entry: StatusBarEntry) {
   class="{opacity(entry)} px-1 {hoverBackground(entry)} {hoverCursor(entry)}"
   title="{tooltipText(entry)}">
   {#if iconClass(entry)}
-    <i class="{iconClass(entry)}" aria-hidden="true"></i>
+    <i class="{iconClass(entry)} not-italic" aria-hidden="true"></i>
```

Comment:
would it be better to use span instead of `i` which is italic tag on which we apply 'not italic' ?

Reply:
I had to remind myself of this old internet debate before the PR: https://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span

`i` is preferred by fa and others, and shorter except in this context; `span` is more correct. I don't feel strongly about either and can easily change if you prefer.

Reply:
I think span is less characters than i+not-italic
so the only advantage is gone so probably go for span only.

Or if you want to stick with i, the no-italic should be set on the upper element ( but not sure in that case it'll not render without italic)

Reply:
Ok, changed to span in new commit.